pkg/oidcclient/login.go

```diff
@@ -44,10 +44,16 @@ import (
 
 const (
 	// minIDTokenValidity is the minimum amount of time that a cached ID token must be still be valid to be considered.
-	// This is non-zero to ensure that most of the time, your ID token won't expire in the middle of a multi-step k8s
+	// This is non-zero to ensure that most of the time, your ID token won't expire in the middle of a multistep k8s
 	// API operation.
 	minIDTokenValidity = 10 * time.Minute
 
+	// minAccessTokenValidity is the minimum amount of time that a cached access token must be still be valid
+	// to be considered.
+	// This is non-zero to ensure that most of the time, your access token won't expire before we submit it for
+	// RFC8693 token exchange.
```

Comment:
Is 10 seconds too short?

Reply:
It should be long enough to immediately submit the token for the token exchange, which is the very next thing that the CLI will do with the token after checking its expiry.

Reply:
Also don't want to make it very long because if an access token is only good for 7 minutes, for example, and we require that it does not expire in the next 5 minutes (instead of 10 seconds), then the CLI is essentially treating it as only a 2-minute token at that point (even though the server said it was a 7-minute token).